docs/user_guide/setup-cmf-worker.md

```diff
@@ -22,8 +46,8 @@ The Unreal Plugin currently must be compiled locally.
 1. Install Visual Studio using the Visual Studio Installer from https://visualstudio.microsoft.com/
 1. Verify your Visual Studio and build tools version are compatible with your version of Unreal by checking the table [here](https://dev.epicgames.com/documentation/en-us/unreal-engine/setting-up-visual-studio-development-environment-for-cplusplus-projects-in-unreal-engine?application_version=5.5)
 1. Under "Individual Components", ensure that the MSVC build tools version selected ("Latest" by default) matches the recommended version in the table. Even though the compatibility guidance may suggest a version "or later", build errors sometimes occur when using a newer version than the one listed as "recommended".
-1. Under �Individual Components�, select a recent .NET Framework SDK (4.6.1 and 4.8.1 have been verified)
-1. Under �Workloads� select �Desktop development with C++�
+1. Under "Individual Components", select a recent .NET Framework SDK (4.6.1 and 4.8.1 have been verified)
```

Comment:
Style nit: for labels and buttons, prefer using **bold test** instead of "quotes: